tests/queries/0_stateless/03152_partition_by_hyperplanes.sql

```diff
@@ -0,0 +1 @@
+SELECT partitionByHyperplanes([Float32(2.0), Float32(3.0)], [[Float32(1.0), Float32(-1.0)], [Float32(-1.0), Float32(2.0)]], [Float32(0.0), Float32(0.0)])
```

Comment:
Is it possible to check the output since the returned string is binary?

Reply:
Yes. You can use `hex` or `bin` functions in the test.